docs/stagenet.md

```diff
@@ -31,21 +31,18 @@ For Linux 64-bit, you can do:
    data. If you skip this step, a new wallet will be created that you can later fund for
    swaps.
 
-4. Create a Goerli wallet. You can do this using Metamask by selecting "Goerli Test Network" from the networks, then creating a new account with "Create account". I'd recommend naming this new account something explicit like `goerli-swap-account`.
+4. Create a Sepolia wallet. You can do this using Metamask by selecting "Sepolia Test Network" from the networks, then creating a new account with "Create account". I'd recommend naming this new account something explicit like `sepolia-swap-account`.
 
 5. Optional: Export the private key for this account by navigating to: three dots in upper right of
    Metamask -> account details -> export private key. Paste this private key into a file
-   named `{DATA_DIR}/eth.key`. If you skip this step, a new goerli wallet will be created for you
-   that you can transfer Goerli ether to or fund directly in the next step.
+   named `{DATA_DIR}/eth.key`. If you skip this step, a new wallet will be created for you
+   that you can transfer Sepolia ether to or fund directly in the next step.
 
-6. Fund your Goerli account using a faucet: 
-- https://goerli-faucet.pk910.de/
-- https://goerlifaucet.com/
-- https://goerli-faucet.mudit.blog/
+6. Fund your Sepolia account using a faucet: 
+- https://sepolia-faucet.pk910.de/
+- https://sepolia.dev/
```

Comment:
https://sepoliafaucet.com/ is the Alchemy maintained faucet that gives out 0.5 SepoliaETH every 24 hours. I think it is the best option if you already have an Alchemy account. The sepolia.dev link is sending me to https://faucet.sepolia.dev/ which is giving a "bad gateway" error for me.